src/app/modules/admin-core/components/home/home.component.ts

```diff
@@ -17,12 +18,22 @@ export class HomeComponent implements OnInit {
   response: boolean = true;
   constructor(private usersService: UsersService, public dialog: MatDialog,
     private KeycloakService: keyCloakService, private _snackBar: MatSnackBar,
-    private router: Router, private route: ActivatedRoute) { }
+    private router: Router, private route: ActivatedRoute,
+    private commonServiceService: CommonServiceService) { }
 
   async ngOnInit() {
-    this.promiseRowData = await this.usersService.getUserRoles();
+    this.promiseRowData = await this.usersService.getUserRoles().catch(e => {
+      // this.KeycloakService.logout();
+      // this.unAuthoriseduser();
+      console.log('@@@@@@@@@@@@@@@', this.promiseRowData);
+      
+    });
+    console.log('@@@@@@@@@@@@@@@', this.promiseRowData);
     if (this.promiseRowData['result']) {
       this.rolesArray = this.promiseRowData['result'].roles;
+    } else {
+      this.response = false
+      this.KeycloakService.logout();
```

Comment:
Did we decide to show unauthorized flow?

Reply:
Changed